snuba/web/split.py

```diff
@@ -128,7 +128,10 @@ def replace_condition(expression: Expression) -> Expression:
     if condition is not None:
         query.set_ast_condition(
             combine_and_conditions(
-                [replace_condition(c) for c in get_first_level_conditions(condition)]
+                [
+                    replace_condition(c)
+                    for c in get_first_level_and_conditions(condition)
```

Comment:
What's the impact (if any) on the time and column splitting?

Reply:
none. This is iso-functional, it executes the same code as before. Before this PR, get_first_level_conditions was returning the top level conditions in AND. Now that method does not exist and we have to specify whether we want to split based on top level AND or top level OR